go/libkb/http.go

```diff
@@ -88,3 +89,18 @@ func HTTPArgsFromKeyValuePair(key string, val HTTPValue) HTTPArgs {
 	ret[key] = val
 	return ret
 }
+
+type ClosingRoundTripper struct {
+	transport *http.Transport
```

Comment:
If we end up leaving this why not have it take in the interface like `InstrumentedRoundTripper` instead of `*http.Transport`?

Reply:
Changed